test/e2e/frontend/cypress/fixtures/libraries/npm-packages.json

```diff
@@ -0,0 +1,30 @@
+{
+    "@flowfuse-KbQ8Wa7EdZ/node-red-example": {
```

Comment:
Should we have hard coded team ids here?

Node sure how this is being used, but just checking that a team with a matching ID will always be created?

Reply:
Good Q - I'm not sure we ever care here, I'm forcing a response of these packages, independent of the provided Team ID, so it could be anything

Reply:
OK, if the tests are expecting that value then it's fine, just appeared strange as I was  going through